packages/aws-cdk-lib/core/lib/analytics-data-source/enums.ts

```diff
@@ -0,0 +1,4028 @@
+/* eslint-disable @cdklabs/no-literal-partition */
+/* eslint-disable quote-props */
+/* eslint-disable @stylistic/comma-dangle */
+/*
+ * Do not edit this file manually. To prevent misconfiguration, this file
+ * should only be modified by an automated GitHub workflow, that ensures
+ * that the ENUMs present in this list
```

Comment:
instead of depend on good intensions, can you update our pr-linter check to make sure that any change to these files are done by a commit from `aws-cdk-automation`, and reject any PR that violate that